lib/gitlab/ci/templates/jobs/gradle/PublishPages.yml

```diff
@@ -0,0 +1,84 @@
+pages_generate:
+  stage: publish
+  rules:
+    - if: '$FORCE_PUBLISH_PAGES =~ /true/i'
+    - if: '$PUBLISH_JAVADOCS_DISABLED =~ /true/i'
+      when: never
+    - if: $CI_COMMIT_TAG
+      when: never
+    - if: $CI_PIPELINE_SOURCE == "merge_request_event"
+      when: never
+    - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
+    - if: '$CI_COMMIT_BRANCH =~ $DEV_OR_RELEASE_REGEX'
+  script:
+    - echo "CI Pipeline Source = '$CI_PIPELINE_SOURCE'"
+    - ./gradlew javadoc -x build -x test
+    - ./gradlew compileJava javadoc -x test
+
+  artifacts:
+    paths:
+      - "build/docs/javadoc"
+      - "**/build/docs/javadoc"
+      - "build/reports"
+      - "**/build/reports"
+
+
+pages:
+  image: "${IMAGE_PREFIX}python:3.11-rc-alpine"
+  stage: deploy
+  variables:
+    ROOT_PROJECT_DIR: $CI_PROJECT_DIR
+  rules:
+    - if: '$FORCE_PUBLISH_PAGES =~ /true/i'
+    - if: '$PUBLISH_JAVADOCS_DISABLED =~ /true/i'
+      when: never
+    - if: $CI_COMMIT_TAG
+      when: never
+    - if: $CI_PIPELINE_SOURCE == "merge_request_event"
+      when: never
+    - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
+    - if: '$CI_COMMIT_BRANCH =~ $DEV_OR_RELEASE_REGEX'
+  script:
+    - echo "CI Pipeline Source = '$CI_PIPELINE_SOURCE'"
+    - rm -rf public
+    - mkdir public
+    - mkdir public/javadocs
+    - mkdir public/coverage
+    - mkdir public/quality
+    - mkdir public/dependency-licenses
+    - mkdir public/dependency-vulnerabilities
+    - mkdir public/source-vulnerabilities
+    - wget https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/make-publish-pages-its-own-job-and-handle-n-tier-gradle-sub-projects/lib/gitlab/ci/templates/references/javadoc/create_aggregate_javadoc_page.py
```

Comment:
Before merging change "make-publish-pages-its-own-job-and-handle-n-tier-gradle-sub-projects" to "release/1.3"